src/components/Publish/_utils.ts

```diff
@@ -203,7 +204,7 @@ export async function createTokensAndPricing(
 
   // TODO: cap is hardcoded for now to 1000, this needs to be discussed at some point
   const ercParams: DatatokenCreateParams = {
-    templateIndex: 2,
+    templateIndex: defaultDatatokenTemplateIndex,
```

Comment:
this is all a bit confusing so let's double check. If I get the thread in here right, we want to continue using the so called "datatoken enterprise template".

We usually refer to this with template index `2`, like we did here before. But this change now sets this to `1` based on the config value in `app.config.js`.

This change here will make new assets published use the default template, `1`. Is this all intended?

Reply:
@alexcos20 ?

Reply:
bump @alexcos20

Reply:
my mistake for spelling error.
We should publish using template 2 (Enterprise Template), because it has a better UX  (buy / order in a single TX)  compared to template 1 (where you need to buy DT first (one tx) + order (2nd tx)

But we need to support both templates when consuming.